tendrl/node_agent/message/logger.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from tendrl.commons.message import Message
+
+
+LOG = logging.getLogger(__name__)
+
+
+class Logger(object):
+    logger_priorities = {"notice": "info",
+                         "info": "info",
```

Comment:
Looks like 'info' and 'notice' are same. why you need both?
Is it done for compatebility with common logging?